examples/nextjs-spa/src/pages/verification.tsx

```diff
@@ -101,19 +67,14 @@ const Verification: NextPageWithLayout = () => {
 
     // If ?flow=.. was in the URL, we fetch it
     if (flowId) {
-      getVerificationFlow(flowId).catch(() => createVerificationFlow(returnTo)) // the flow might be expired, so we create a new one
+      getFlow(flowId).catch(() => createFlow(returnTo)) // the flow might be expired, so we create a new one
       return
     }
 
     // Otherwise we initialize it
-    createVerificationFlow(returnTo)
-  }, [
-    getVerificationFlow,
-    createVerificationFlow,
-    flowId,
-    returnTo,
-    router.isReady,
-  ])
+    createFlow(returnTo)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'm not sure if we should disable linter rules like this.
Since the functions are memoized through the `useCallback` function, it should be alright to add them to this dependency list.

Reply:
For all `useEffect` I removed the eslint disable, but did not add those dependencies.

Adding those would require a different kind of flow than this was originally designed to. Currently, if you add those, then for example when first render `useEffect` > `createFlow` > URL params change > `useEffect` > `getFlow`

I think it's better now to leave those without dependencies even when that too has some unwanted side effects.